pages/blog/2023-Q1-docs-report.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "AsyncAPI Documentation Report - Q1 2023"
+date: 2023-04-08T06:00:00+01:00
+type: Engineering
+tags:
+  - Documentation
+  - Communication
+cover: /img/posts/gsod-2022/SeasonofDocs_Logo.webp
```

Comment:
Accepting ideas/recommendations for best cover photo idea 😄

Reply:
From my perspective, since the blog covers all aspects of AsyncAPI documentation then the banner should definitely have that Ale's-documentation style vibes.  😊